docs/outlook/smart-alerts-onmessagesend-walkthrough.md

```diff
@@ -624,14 +693,16 @@ If you implemented the optional steps to customize the **Don't Send** button or
 
         ![Dialog with a customized Don't Send button requesting the user to add an attachment to the message.](../images/outlook-smart-alerts-custom-button.png)
 
+        //TODO - Add screenshot with Take Action implementation.
```

Comment:
did you intend to add this screenshot now or later?

Reply:
This will be added later before we publish.